tests/integration/test_ec2.py

```diff
@@ -277,3 +304,76 @@ def test_describe_vpc_endpoints_with_filter(self, ec2_client):
 
         # clean up
         ec2_client.delete_vpc(VpcId=vpc_id)
+
+    @pytest.mark.aws_validated
+    @pytest.mark.parametrize("id_type", ["id", "name"])
+    def test_modify_launch_template(self, ec2_client, create_launch_template, id_type):
+        launch_template_result = create_launch_template(f"template-with-versions-{short_uid()}")
+        template = launch_template_result["LaunchTemplate"]
+
+        # call the API identifying the template wither by `LaunchTemplateId` or `LaunchTemplateName`
+        kwargs = (
+            {"LaunchTemplateId": template["LaunchTemplateId"]}
+            if (id_type == "id")
+            else {"LaunchTemplateName": template["LaunchTemplateName"]}
+        )
```

Comment:
This might be more readable?

```suggestion
    @pytest.mark.parametrize("id_type", ["LaunchTemplateId", "LaunchTemplateName"])
    def test_modify_launch_template(self, ec2_client, create_launch_template, id_type):
        launch_template_result = create_launch_template(f"template-with-versions-{short_uid()}")
        template = launch_template_result["LaunchTemplate"]

        # call the API identifying the template wither by `LaunchTemplateId` or `LaunchTemplateName`
        kwargs = {id_type: template[id_type]}
```

Reply:
Interesting though. I tend to go with a more descriptive code style in this situations, as I think it communcates the intent quicker with reading less code. On the other hand there is some code duplication here of course. I'm also not very proficient with python yet, so happy to go with your decision on this!